presentation/main_view.py

```diff
@@ -10,10 +11,12 @@
 from presentation.matrix_view import make_mesh_test_matrix_layout
 
 
-def make_page_layout(mesh: MeshResults, config: Config) -> html.Div:
+def make_page_layout(mesh: MeshResults, results_timestamp: datetime, config: Config) -> html.Div:
+    timestamp = results_timestamp.strftime("%H:%M:%S")
+    header = f"Demo WebApp main page - test results timestamp {timestamp}"
```

Comment:
Eventually, all texts shown in UI need to configurable too, because they may need to be localized.

Reply:
This will be subject to a separate PR